repository_service_tuf/cli/admin/import_targets.py

```diff
@@ -0,0 +1,201 @@
+                    "last_update": datetime.now(),
+                }
+            )
+
+    return rstuf_db_data
+
+
+def _import_csv_to_rstuf(
+    db_client: Connection,
+    rstuf_table: Table,
+    csv_files: List[str],
+    succinct_roles: SuccinctRoles,
+) -> None:
+    for csv_file in csv_files:
+        console.print(f"Import status: Loading data from {csv_file}")
+        rstuf_db_data = _parse_csv_data(csv_file, succinct_roles)
+        console.print(f"Import status: Importing {csv_file} data")
+        try:
+            db_client.execute(rstuf_table.insert(), rstuf_db_data)
+        except IntegrityError:
+            raise click.ClickException(
+                "Import status: ABORTED due duplicated targets. "
+                "CSV files must to have unique targets (path). "
+                "No data added to RSTUF DB."
+            )
+        console.print(f"Import status: {csv_file} imported")
+
+
+def _get_succinct_roles(metadata_url: str) -> SuccinctRoles:
+    response = request_server(metadata_url, "1.bin.json", Methods.get)
+    if response.status_code == 404:
+        raise click.ClickException("RSTUF Metadata Targets not found.")
+
+    json_data = json.loads(response.text)
+    targets: Metadata[Targets] = Metadata.from_dict(json_data)
+    if targets.signed.delegations is None:
+        raise click.ClickException("Failed to get Targets Delegations")
+
+    targets_delegations: Delegations = targets.signed.delegations
+
+    if targets_delegations.succinct_roles is None:
+        raise click.ClickException("Failed to get Targets succinct roles")
+
+    return targets_delegations.succinct_roles
+
+
+@admin.command()
+@click.option(
+    "-metadata-url",
+    required=True,
+    help="RSTUF Metadata URL i.e.: http://127.0.0.1 .",
+)
+@click.option(
+    "-db-uri",
+    required=True,
+    help="RSTUF DB URI. i.e.: postgresql://postgres:secret@127.0.0.1:5433",
+)
+@click.option(
+    "-csv",
+    required=True,
+    multiple=True,
+    help=(
+        "CSV file to import. Multiple -csv parameters are allowed. "
+        "See rstuf CLI guide for more details."
+    ),
+)
+@click.option(
+    "--skip-publish-targets",
+    is_flag=True,
+    help="Skip publishing targets in TUF Metadata.",
+)
+@click.pass_context
+def import_targets(
+    context: Any,
+    metadata_url: str,
+    db_uri: str,
+    csv: List[str],
+    skip_publish_targets: bool,
+):
+    """
+    Import targets to RSTUF from exported CSV file.
+    """
+    settings = context.obj["settings"]
+    server = settings.get("SERVER")
+    token = settings.get("TOKEN")
+    if server and token:
+        token_access_check = is_logged(server, token)
+        if token_access_check.state is False:
+            raise click.ClickException(
+                f"{str(token_access_check.data)}"
+                "\n\nTry re-login: 'Repository Service for TUF admin login'"
+            )
+    else:
+        raise click.ClickException("Login first. Run 'rstuf admin login'")
```

Comment:
This part still seems to me that IT is checked with `headers = get_headers(settings)` ([here](https://github.com/vmware/repository-service-tuf-cli/blob/1320eb54b127d35c438bb7f66761906b2661efee/repository_service_tuf/helpers/api_client.py#L85-L91)), why do we need it separately here?